tools/build-signer.sh

```diff
@@ -0,0 +1,110 @@
+#!/bin/bash
```

Comment:
It looks like a lot of stuff was copied from https://github.com/mobilecoinofficial/full-service/blob/zd/transaction-signer-lib-pyo3/tools/build-fs.sh
What is the reason for not having a single script that builds all the binaries in this repo?